docs/tutorials/chunked.md

```diff
@@ -0,0 +1,90 @@
+# Chunking Data for Zarr Storage
+
+This tutorial will provide an example of writing chunked data to a Zarr storage device.
+
+Zarr has additional capabilities relative to the basic storage devices, namely _chunking_, _compression_, and _multiscale storage_. To enable _chunking_, set the attributes in an instance of the `ChunkingProperties` class. You can learn more about the Zarr capabilities in `Acquire` [here](https://github.com/acquire-project/acquire-driver-zarr).
+
+## Configure `Runtime`
+To start, we'll create a `Runtime` object and configure the streaming process, selecting `Zarr` as the storage device to enable chunking the data.
+
+```python
+import acquire
+
+# Initialize a Runtime object
+runtime = acquire.Runtime()
+
+# Initialize the device manager
+dm = runtime.device_manager()
+
+# Grab the current configuration
+config = runtime.get_configuration() 
+
+# Select the radial sine simulated camera as the video source
+config.video[0].camera.identifier = dm.select(acquire.DeviceKind.Camera, "simulated: radial sin") 
+
+# Set the storage to Zarr to have the option to save multiscale data
+config.video[0].storage.identifier = dm.select(acquire.DeviceKind.Storage, "Zarr")
+
+# Set the time for collecting data for a each frame
+config.video[0].camera.settings.exposure_time_us = 1e4  # 10 ms

```

Comment:
```suggestion
config.video[0].camera.settings.exposure_time_us = 5e4  # 50 ms
```

I've had trouble with dropping frames and an overall slowdown of the runtime at 10 ms. It's a bug that we need to fix, but until then this will probably be a better user experience.